src/fields/validations.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { text, textarea, password, select, point, number } from './validations';
 import { ValidateOptions } from './config/types';
+import payload from '..';
```

Comment:
Payload is exposed within `validate` functions automatically on the server. There is no need to import it here as it won't have any effect in the client UI anyway. Getting access to `payload` should be done within each validate function's `options` arg. We should remove this import and access `payload` from `options`, only if it is present (aka we are on the server).

Reply:
Will do